torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -1048,31 +1068,41 @@ at::Tensor PythonArgs::tensor_slow(int i) {
   return tensor;
 }
 
-at::Scalar PythonArgs::scalar_slow(int i) {
-  if (traceable && jit::tracer::isTracing() && THPVariable_Check(args[i])) {
-    auto& var = THPVariable_Unpack(args[i]);
-    jit::tracer::ArgumentStash::stashValue(
-        signature.params[i].name, idx, var, jit::NumberType::get());
-  }
 
+template <typename T>
+at::Scalar PythonArgs::get_scalar(T arg) {
```

Comment:
thanks

Reply:
Not sure why you need to template this though, doesn't this always get a PyObject or py::handle or something similarly convertible?

Reply:
I hope a refcount bump isn't happening here either... would be easier to tell if this wasn't templated.

Reply:
You are right. We dont need this at all